modules/machineset-yaml-alibaba-usage-stats.adoc

```diff
@@ -0,0 +1,96 @@
+// * machine_management/creating_machinesets/creating-machineset-alibaba.adoc
+
+ifeval::["{context}" == "creating-infrastructure-machinesets"]
+:infra:
+endif::[]
+
+:_content-type: REFERENCE
+[id="machineset-yaml-alibaba-usage-stats_{context}"]
+= Machine set parameters for Alibaba Cloud usage statistics
+
+The default machine sets that the installer creates for Alibaba Cloud clusters include nonessential tag values that Alibaba Cloud uses internally to track usage statistics. These tags are populated in the `securityGroups`, `tag`, and `vSwitch` parameters of the `spec.template.spec.providerSpec.value` list.
+
+When creating machine sets to deploy additional machines, you must include the required Kubernetes tags. The usage statistics tags are applied by default, even if they are not specified in the machine sets you create. You can also include additional tags as needed.
+
+The following YAML snippets indicate which tags in the default machine sets are optional and which are required.
+
+.Tags in `spec.template.spec.providerSpec.value.securityGroups`
+[source,yaml]
+----
+spec:
+  template:
+    spec:
+      providerSpec:
+        value:
+          securityGroups:
+          - tags:
+            - Key: kubernetes.io/cluster/<infrastructure_id> <1>
+              Value: owned
+            - Key: GISV
+              Value: ocp
+            - Key: sigs.k8s.io/cloud-provider-alibaba/origin <1>
+              Value: ocp
+            - Key: Name
+              Value: <infrastructure_id>-sg-<role> <2>
+            type: Tags
+----
+<1> Optional: This tag is applied even when not specified in the machine set.
+<2> Required.
++
+where:
++
+* `<infrastructure_id>` is the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster.
+* `<role>` is the node label to add.
+
+.Tags in `spec.template.spec.providerSpec.value.tag`
+[source,yaml]
+----
+spec:
+  template:
+    spec:
+      providerSpec:
+        value:
+          tag:
+          - Key: kubernetes.io/cluster/<infrastructure_id> <2>
+            Value: owned
+          - Key: GISV <1>
+            Value: ocp
+          - Key: sigs.k8s.io/cloud-provider-alibaba/origin <1>
+            Value: ocp
+----
+<1> Optional: This tag is applied even when not specified in the machine set.
+<2> Required.
++
+where `<infrastructure_id>` is the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster.
+
+.Tags in `spec.template.spec.providerSpec.value.vSwitch`
+[source,yaml]
+----
+spec:
+  template:
+    spec:
+      providerSpec:
+        value:
+          vSwitch:
+            tags:
+            - Key: kubernetes.io/cluster/<infrastructure_id> <1>
+              Value: owned
+            - Key: GISV <1>
+              Value: ocp
+            - Key: sigs.k8s.io/cloud-provider-alibaba/origin <1>
+              Value: ocp
+            - Key: Name
+              Value: <infrastructure_id>-vswitch-<region>-<zone> <2>
```

Comment:
It should be more appropriate to change `<infrastructure_id>-vswitch-<region>-<zone>`  to `<infrastructure_id>-vswitch-<zone>` , for example, the label of the VSwitch of the cluster I created, `shtest008-pwqbx-vswitch-cn-hangzhou- j`, The `cn-hangzhou-j` is an Availability Zone ID. For more details, please refer to [DescribeZones](https://www.alibabacloud.com/help/en/doc-detail/25610.html)